release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -143,6 +143,83 @@ In earlier versions of {product-title}, Ironic container images used {op-system-
+You can create the agent ISO image with the following IP address configurations:
+
+* IPv4
+* IPv6
+* IPv4 and IPv6 in parallel (dual-stack)
+
+For more information, see xref:../installing/installing_with_agent_based_installer/installing-with-agent-based-installer.html#installing-ocp-agent_installing-with-agent-based-installer[Dual and single IP stack clusters].
+
+[id="ocp-4-12-agent-based-install-mce"]
+
+==== The agent deployed {product-title} cluster can be used as a hub cluster
+
+You can install the multicluster engine for Kubernetes(MCE) Operator and deploy a hub cluster with the Agent-based Installer.
+
+// TODO doc PR not merged see xref:..
+
+[id="ocp-4-12-agent-based-install-validations"]
+==== Agent-based installation performs installation validations
+
+The Agent-based Installer performs validations on:
+
+* Installation image generation: The user-provided manifests are checked for validity and compatibility.
+* Installation: The installation service checks the hardware available for installation and emits validation events that can be retrieved with the `openshift-install agent wait-for` subcommands.
+
+For more information, see xref:../installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.html#validations-before-agent-iso-creation_preparing-to-install-with-agent-based-installer[Installation validations].
+
+[id="ocp-4-12-agent-based-install-static-networking"]
+==== Configure static networking in a Agent-based installation
+With the Agent-based {product-title} Installer, you can configure static IP addresses for IPv4, IPv6, or dual-stack (both IPv4 and IPv6) for all the hosts prior to creating the agent ISO image. You can add the static addresses to the `hosts` section of the `agent-config.yaml` file or in the `NMStateConfig.yaml` file if you are using the ZTP manifests.
+Note that the configuration of the addresses must follow the syntax rules for nmstate as described in link:https://nmstate.io/examples.html[NMState state examples].
+
+For more information, see xref:../installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.html#agent-install-networking_preparing-to-install-with-agent-based-installer[About networking].
+
+[id="ocp-4-12-agent-based-install-automated-deployment"]
+==== CLI based automated deployment in an Agent-based installation
+With the Agent-based {product-title} Installer, you can define your installation configurations, generate an ISO for all the nodes, and then have an unattended installation by booting the target systems with the generated ISO.
+For more information, see xref:../installing/installing_with_agent_based_installer/installing-with-agent-based-installer.html#installing-with-agent-based-installer[Installing a {product-title} cluster with the Agent-based Installer].
+
+[id="ocp-4-12-agent-based-install-host-specific"]
+==== Agent-based Installation supports host specific configuration at the instalation time
```

Comment:
This was the only instance of "Agent-based Installation" where "Installation" was init-capped. Maybe lowercase to match? I'm assuming also that for "Agent-based Installer", it is intentional to init-cap "Installer".

Reply:
Good catch, it should be in small caps